addons/mail/tests/test_mail_template.py

```diff
@@ -45,6 +63,30 @@ def test_mail_compose_message_content_from_template_mass_mode(self):
         self.assertEqual(values[self.partner_employee.id]['subject'], '6', 'We must trust mail template values')
         self.assertIn('13', values[self.partner_employee.id]['body_html'], 'We must trust mail template values')
 
+        # Send a mail with MSO conditionals.
+        mail_compose_message = self.env['mail.compose.message'].create({
```

Comment:
I would make a dedicated unit test instead of adding a test about comments in a test about rendering security. We could even do two tests (one with comments in composer, one with comments coming from template).

Reply:
Where would I put these two then? One in test_mail_template in a new function, and one in test_mail_full_composer in a new function?

Reply:
Well to avoid too much split, maybe a class ``TestMailComposer``  in test_mail_full_composer.py file, whose purpose is to test basics of composer. The ``TestMailFullComposer`` class is for UI (tour), here we can add another class for classic composer unit tests. I would put those two tests in that new class, so that tests are located in a single file. We could probably merge both files (mail_template and mail_full_composer) afterwards, but no need to do it here as it is outside the scope of this task :) .

Reply:
I see there's already a `test_mail_composer.py`, in the `test_mail` module :O Wouldn't it fit there instead then?

Reply:
test_mail is more for integration tests on custom models (more testing the "mail.thread" mixin and all its features). Basic tests are done in mail (UI). It is a bit the bordel between those two modules.. I think those tests belong more to mail as they do not depend really on any mail.thread implementation.

Reply:
Ok done. Was a bit trickier than expected but hopefully it's good now :)